test/unit/tests-api/actions-builder.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const q = require('q');
+
+const ActionsBuilder = require('../../../lib/tests-api/actions-builder');
+const util = require('../../util');
+
+describe('tests-api/actions-builder', () => {
+    const sandbox = sinon.sandbox.create();
+    const browser = util.makeBrowser();
+
+    const createActionsBuilder = (actions) => new ActionsBuilder(actions || []);
+
+    const runAction = (method, browser, postActions) => {
+        const state = util.makeStateStub();
+
+        createActionsBuilder(state.actions)[method]();
+        return state.actions[0](browser, postActions);
+    };
+
+    afterEach(() => sandbox.restore());
+
+    describe('changeOrientation', () => {
+        beforeEach(() => {
+            sandbox.stub(browser, 'getOrientation').returns(q());
+            sandbox.stub(browser, 'setOrientation').returns(q());
+        });
+
+        it('should throw in case of passed arguments', () => {
```

Comment:
`should throw in case of passed arguments` => `should throw error in case of passed invalid arguments`